Queries.sql

```diff
@@ -0,0 +1,83 @@
+SELECT
```

Comment:
A description about each query would be helpful.

Reply:
Added one line comment about each query